frontend/lint-staged.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  "*.{js,jsx,ts,tsx,md,html,css}": "npx prettier --write",
+  "*.{js,jsx,ts,tsx,md,html,css,yml}": "npx prettier --write",
```

Comment:
When does this run, and will it find corpora-data-portal/backend/config/corpora-api.yml?

Reply:
this runs on `git commit`, and it checks all the files being committed, so it should catch `corpora-data-portal/backend/config/corpora-api.yml` 🙆‍♂️